src/style/app.css

```diff
@@ -35,7 +35,7 @@
 
   /* Layout */
   .freq-container {
-    @apply mx-auto w-full sm:px-[30px] md:px-[40px] lg:max-w-[100vw] lg:px-[80px] xl:max-w-page;
```

Comment:
I think we still want `mx-auto` to center content.

Reply:
I think this was a rebase conflict error; I'll put it back.